WebJobs.Script.sln

```diff
@@ -1,8 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 17
```

Comment:
As it's a comment, I don't think it matters for parsing, so I added a VS2017 as well